Firestore/Example/Tests/Util/FSTHelpers.h

```diff
@@ -229,9 +227,10 @@ FSTViewSnapshot *_Nullable FSTTestApplyChanges(FSTView *view,
 FSTSetMutation *FSTTestSetMutation(NSString *path, NSDictionary<NSString *, id> *values);
 
 /** Creates a patch mutation for the document key at the given path. */
-FSTPatchMutation *FSTTestPatchMutation(NSString *path,
-                                       NSDictionary<NSString *, id> *values,
-                                       NSArray<FSTFieldPath *> *_Nullable updateMask);
+FSTPatchMutation *FSTTestPatchMutation(
+    const absl::string_view path,
+    const std::map<std::string, std::string> &values,
```

Comment:
Absent any special magical template support this likely has to be `std::map<std::string, model::FieldValue>` so that we can continue to express mutations that mutate non-string values.

Likely you'll want to just leave this alone (reverting this parameter to `NSDictionary<NSString *, id>` until you're ready to tackle the conversion of FSTFieldValue to FieldValue.